RUM Auto-instrumentation updates (#28233)
DataDog/documentation: content/en/real_user_monitoring/browser/setup/server/_index.md

---
title: Browser Monitoring Auto-Instrumentation (Server-side)
beta: true
code_lang: server
type: multi-code-lang
code_lang_weight: 2
further_reading:
- link: '/real_user_monitoring/explorer/'
  tag: 'Documentation'
  text: 'Learn about the RUM Explorer'
- link: '/logs/log_collection/javascript/'
  tag: 'Documentation'
  text: 'Learn about the Datadog Browser SDK for Logs'
---

## Overview

RUM Auto-Instrumentation works by injecting the RUM Browser SDK into the HTML responses being served through a web server or proxy.

## Getting started

<div class="alert alert-info">RUM Auto-Instrumentation is in Preview. To request support for a web server that is not listed here, <a href="https://www.datadoghq.com/private-beta/rum-sdk-auto-injection/">fill out this form.</a></div>

> Follow the relevant documentation for automatic instrumentation approach to learn more:

Reviewer: ```suggestion
Follow the documentation for your web server type to learn more:
```

---
title: Browser Monitoring Auto-Instrumentation (Server-side)
beta: true
code_lang: server
type: multi-code-lang
code_lang_weight: 2
further_reading:
- link: '/real_user_monitoring/explorer/'
  tag: 'Documentation'
  text: 'Learn about the RUM Explorer'
- link: '/logs/log_collection/javascript/'
  tag: 'Documentation'
  text: 'Learn about the Datadog Browser SDK for Logs'
---

## Overview

RUM Auto-Instrumentation works by injecting the RUM Browser SDK into the HTML responses being served through a web server or proxy.

## Getting started

### Remotely configure your RUM applications in Datadog

Remotely manage your RUM application configurations directly from Datadog, after having them auto-instrumented with the RUM SDK.
If you want to manage RUM configs in your web server files instead, see [Non-remote managed applications](#non-remote-managed-applications).

> Configurations that can be remotely managed are as follows:

Reviewer: ```suggestion
The following configurations can be remotely managed:
```

- Session sampling rate
- Session Replay sampling rate
- Session Replay privacy setting
- Action name privacy setting
- Environment (attribute)
- Service name (attribute)

> 

Reviewer: Can we add some type of lead-in sentence before we get to the numbered section here? Going from bullets to the numbers is a little confusing. Are these the official steps for "Remotely configure your RUM applications in Datadog"?

> 

Author: To remotely configure your RUM application:

> 

Author: ```suggestion

To remotely configure your RUM application:
```

> 1. Make sure you have [Remote Configuration][42] enabled in Datadog.

Reviewer: Broken link?

> 1. In Datadog, navigate to the [**Digital Experience > Add an Application Page**][1] and select the JavaScript (JS) application type.

Reviewer: Broken link here as well.
